commit fb6b8983f1139d09878f39693a485eca2832f6d5
Author: Paul Wise <pabs3@bonedaddy.net>
Date:   Fri May 9 16:29:19 2008 +0800

    Replace non-free TexFont with a TexFont-compatible wrapper for plib.

diff --git a/src/CHROMIUM.pro b/src/CHROMIUM.pro
index cedc211..6e78557 100644
--- a/src/CHROMIUM.pro
+++ b/src/CHROMIUM.pro
@@ -38,7 +38,7 @@ isEmpty(GL_LIBS) {
 
 	
 
-LIBS		+= -lSDL_mixer -lglpng $(GL_LIBS) $(AL_LIBS) $(SDL_LIBS) $(SMPEG_LIBS) $(VORBIS_LIBS)
+LIBS		+= -lplibfnt -lplibul -lSDL_mixer -lglpng $(GL_LIBS) $(AL_LIBS) $(SDL_LIBS) $(SMPEG_LIBS) $(VORBIS_LIBS)
 
 ##-- Debug --
 #CONFIG		+= warn_on debug
diff --git a/src/MainGL.cpp b/src/MainGL.cpp
index 1120088..19152ef 100644
--- a/src/MainGL.cpp
+++ b/src/MainGL.cpp
@@ -97,21 +97,17 @@ int MainGL::initGL()
 //----------------------------------------------------------
 void MainGL::loadTextures()
 {
-	GLuint	texobj;
 	game->texFont = txfLoadFont( dataLoc("fonts/space.txf") );
 	if(!game->texFont)
 	{
 		fprintf(stderr, "\nERROR loading texture font. Check data path and try again.\n\n");
 		exit(1);
 	}
-	glGenTextures(1, &texobj);
-	txfEstablishTexture(game->texFont, texobj, GL_FALSE);
 }
 
 //----------------------------------------------------------
 void MainGL::deleteTextures()
 {
-	glDeleteTextures(1, &game->texFont->texobj);
 	txfUnloadFont(game->texFont);
 	game->texFont = 0;
 }
